node/chainSimulator/components/configLoaders.go

```diff
@@ -0,0 +1,19 @@
+package components
+
+import (
+	"os"
+
+	"github.com/pelletier/go-toml"
+)
+
+// LoadConfigFromFile will try to load the config from the specified file
+func LoadConfigFromFile(filename string, config interface{}) error {
```

Comment:
remove this and use mx-chain-go-core/core.LoadTomlFile ?

Reply:
done